docs/source/en/using-diffusers/loading.mdx

```diff
@@ -23,31 +23,50 @@ In the following we explain in-detail how to easily load:
 
 ## Loading pipelines
 
-The [`DiffusionPipeline`] class is the easiest way to access any diffusion model that is [available on the Hub](https://huggingface.co/models?library=diffusers). Let's look at an example on how to download [CompVis' Latent Diffusion model](https://huggingface.co/CompVis/ldm-text2im-large-256).
```

Comment:
Update the example here to use Stable Diffusion as the main example instead of LDM. We used LDM back then because of gated access but it's no longer the case.

Reply:
Makes total sense!